skunkworks_crow/src/main/AndroidManifest.xml

```diff
@@ -78,11 +81,13 @@
             android:name=".provider.InstanceMapProvider"
             android:authorities="org.odk.share.provider.odk.map" />
 
-        <activity android:name=".activities.SendFormsActivity"
-                  android:parentActivityName=".activities.MainActivity">
+        <activity
+            android:name=".activities.SendFormsActivity"
+            android:configChanges="orientation|screenSize"
```

Comment:
I think you should highlight this code as so many changes related to the code style, hard for reviewers to find what truly changed.

Reply:
That is the reason I added the line in PR template

Reply:
Sure, making a self comment in the code base would be better.

Reply:
@huangyz0918 like that I made self comments in PR #240 but @shobhitagarwal1612 asked to remove those comments due to code style will not make better

Reply:
I mean, making comment in Github review panel.

Reply:
@Chromicle I don't know if its the best solution or not, that's the reason I didn't add it in the first place.
It would be helpful if you can go through the pros and cons of using it, and then update your PR description to back up the changes that you had done.
For ref. https://stackoverflow.com/a/7990543